_source/logzio_collections/_metrics-sources/app-service-azure-metrics.md

```diff
@@ -0,0 +1,155 @@
+
+##### Log in to Azure with the CLI
+
+Run this command in the command line:
+
+```yml
+az login
+```
+
+To complete the process, sign in through your browser.
+
+##### Get your subscription details
+
+Run this command:
+
+```yml
+az account show | grep '"id"\\|"tenantId"'
+```
+
+The response shows the subscription ID (the `id` field)
+and tenant ID (the `tenantId` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```
+"id": "d94b1fba-0289-557e-b063-0b6bfc1bdca0",
+"tenantId": "9ae0715a-0689-56e8-bb88-2b22f1fa7299",
+```
+
+##### Create a new Azure AD application
+
+The Metricbeat configuration needs to include credentials for an AD application with reader permissions.
+
+If you don't already have an AD application with reader permissions,
+run this command to create one.
+Replace `<<SUBSCRIPTION-ID>>` with the `id` value from step 2:
+
+```
+az ad sp create-for-rbac --role reader \\
+--scopes /subscriptions/<<SUBSCRIPTION-ID>> \\
+-n logzio-metricbeat \\
+| grep '"appId"\\|"password"'
+```
+
+The response shows the client ID (the `appId` field)
+and client secret (the `password` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```
+"appId": "3dcdf53e-f93f-5902-8df2-235c8635aa4d",
+"password": "e6ab6d24-4907-5d11-a132-a171ef55355d",
+```
+
+##### Download the [Logz.io](http://logz.io/) public certificate
+
+
+
+For HTTPS shipping, download the [Logz.io](http://logz.io/) public certificate to your certificate authority folder.
+You'll need to run this command on the server that hosts Metricbeat:
+
+```
+sudo curl <https://raw.githubusercontent.com/logzio/public-certificates/master/TrustExternalCARoot_and_USERTrustRSAAAACA.crt> --create-dirs -o /etc/pki/tls/certs/COMODORSADomainValidationSecureServerCA.crt
+```
+
+##### (_Optional_) Disable the system module
+
+By default, Metricbeat ships system metrics from its host.
+Disable this module so you don't unintentionally send host metrics:
+
+```yml
+sudo metricbeat modules disable system
+```
+
+##### Add [Logz.io](http://logz.io/) configuration
+
+Now you'll set up the Metricbeat
+to collect metrics from Azure App Service.
+
+You'll need to replace the values surrounded by angle brackets
+`<< >>`
+using the parameters below the code block. 👇
+
+```
+metricbeat.modules:
+- module: azure
+  metricsets:
+  - monitor
+  enabled: true
+  period: 300s
+  client_id: '${AZURE_CLIENT_ID:"<<CLIENT-ID>>"}' # `appId` from step 3
+  client_secret: '${AZURE_CLIENT_SECRET:"<<CLIENT-SECRET>>"}' # `password` from step 3
+  tenant_id: '${AZURE_TENANT_ID:"<<TENANT-ID>>"}' # `tenantId` from step 2
+  subscription_id: '${AZURE_SUBSCRIPTION_ID:"<<SUBSCRIPTION-ID>>"}' # `id` from step 2
+  refresh_list_interval: 600s
+	resources:
+    # 👇 Duplicate this code block for each resource type whose metrics you want to ship.
+		- resource_query: "resourceType eq 'Microsoft.Web/sites'"
```

Comment:
There are extra spaces in this line compared to the original.
Also, instead of 
namespace: "Microsoft.EventHub/namespaces"
you have:
namespace: "Microsoft.Web/sites"